notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<p>Wikipedia is fine, especially since it's not the only resource you provide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>